src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -1699,9 +1750,7 @@ def _render_user_profile_preamble() -> str:
     )
 
 
-def _render_generated_preamble(
-    relative_path: str, frontmatter: dict[str, object]
-) -> str:
+def _render_generated_preamble(relative_path: str, frontmatter: dict[str, object]) -> str:
```

Ruff (0.15.12)

[warning] Unused function argument: `relative_path`

(ARG001)